packages/cactus-plugin-ledger-connector-quorum/src/test/typescript/integration/plugin-ledger-connector-quorum/deploy-contract/openapi/openapi-validation.test.ts

```diff
@@ -394,17 +396,18 @@ test(testCase, async (t: Test) => {
         fake: 4,
       };
       await apiClient.runTransactionV1(parameters as RunTransactionRequest);
-    } catch (e) {
+    } catch (err) {
+      const e = err as AxiosError<{ path: string }[]>;
       t2.equal(
-        e.response.status,
+        e?.response?.status,
         400,
         `Endpoint ${fRun} with fake=4: response.status === 400 OK`,
       );
-      const fields = e.response.data.map((param: any) =>
+      const fields = e?.response?.data.map((param) =>
```

Comment:
@aldousalvarez

Casting the whole exception is also a good solution, but in the interest of keeing the diff smaller on account of conflicts that I foresee with other PRs, please fix the linter warning with a one liner like this: 

```typescript
const fields = e.response.data.map((param: { path: string }) =>
```